linux/net/rina/dtp-ps-default.c

```diff
@@ -285,9 +288,22 @@ default_sender_inactivity_timer(struct dtp_ps * ps)
                         LOG_ERR("Coudln't flush cwq");
                         return -1;
                 }
+                dt_sv_window_closed_set(dt, false);
         }
         rcu_read_unlock();
 
+        init_credit = dtcp_initial_credit(cfg);
+        max_sent    = dtp_sv_max_seq_nr_sent(dtp);
+        snd_rt_win  = dtcp_snd_rt_win(dtcp);
+        next_send   = dtp_sv_last_nxt_seq_nr(dtp);
+
+        LOG_DBG("Current values:\n\tinit_credit: %u "
+                "max_sent: %u snd_rt_win: %u next_send: %u",
```

Comment:
IF you do this you'll end up with two LOGs which can be separated by other log statements. It has to be atomic.